tests/datasets/test_malaria.py

```diff
@@ -0,0 +1,7 @@
+import keras_rcnn.datasets.malaria
+
+def test_load_data():
+    training, test = keras_rcnn.datasets.malaria.load_data()
```

Comment:
We should cache the `~/.keras` directory on Travis to minimize re-downloading.

https://docs.travis-ci.com/user/caching/#Arbitrary-directories